src/to-do-list.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Kindly remove all disabled Linters, Linting improves code readability and removes errors 🙂